ius-release.spec

```diff
@@ -56,7 +61,31 @@ install -D -p -m 0644 %{S:23} %{buildroot}%{_sysconfdir}/yum.repos.d/ius-archive
 %config(noreplace) %{_sysconfdir}/yum.repos.d/ius-archive.repo
 
 
+%package -n ius-mock-cfg
+Version:	1
+Summary:	IUS configuration file for use with mock
+BuildArch:	noarch
+
+Requires:	mock
+
+
+%description -n	ius-mock-cfg
+%{summary}
+
+
+%files -n ius-mock-cfg
+%if %{defined el6}
+%config(noreplace) %{_sysconfdir}/mock/ius-6-x86_64.cfg
+%endif
+%if %{defined el7}
+%config(noreplace) %{_sysconfdir}/mock/ius-7-x86_64.cfg
+%endif
```

Comment:
Let's streamline this to `%config(noreplace) %{_sysconfdir}/mock/ius-*-x86_64.cfg`, no need to have the conditionals.